src/net/codec.rs

```diff
@@ -878,6 +878,20 @@ impl StacksMessageCodec for RelayData {
     }
 }
 
+impl StacksMessageCodec for Attachment {
```

Comment:
Is this used anywhere?  Also, why communicate the hash at all, when it can be derived from the content?

Reply:
No it's dead code, good catch. Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/dab25f4fef571e4a55d2075ec19a85306eeca728